resolwe_bio/management/commands/generate_de.py

```diff
@@ -0,0 +1,157 @@
+import gzip
```

Comment:
Add `from __future__ import absolute_import, division, print_function, unicode_literals` to ensure compatibility with Python 2 and 3.